src/huggingface_hub/fastai_utils.py

```diff
@@ -351,19 +355,49 @@ def from_pretrained_fastai(
     return load_learner(os.path.join(storage_folder, "model.pkl"))
 
 
+@_deprecate_positional_args(version="0.12")
+@_deprecate_arguments(
+    version="0.12",
+    deprecated_args={
+        "git_user",
+        "git_email",
+    },
+)
 @validate_hf_hub_args
 def push_to_hub_fastai(
+    # NOTE: New arguments since 0.9
     learner,
     repo_id: str,
     commit_message: Optional[str] = "Add model",
     private: bool = False,
```

Comment:
```suggestion
def push_to_hub_fastai(
    # NOTE: New arguments since 0.9
    learner,
    *,
    repo_id: str,
    commit_message: Optional[str] = "Add model",
    private: bool = False,
```